src/styles/annotator/adder.scss

```diff
@@ -97,20 +97,12 @@ $adder-transition-duration: 80ms;
   flex-direction: row;
 }
 
-.annotator-adder-actions:hover .annotator-adder-actions__button {
-  color: var.$grey-semi !important;
-}
-
-.annotator-adder-actions:hover .annotator-adder-actions__label {
-  color: var.$grey-semi !important;
-}
-
 .annotator-adder-actions__button {
   box-shadow: none;
   font-family: h;
   font-size: 18px;
-  background: transparent !important;
-  color: var.$grey-mid !important;
+  background: transparent;
```

Comment:
This is a nit, but it's nice to have layout rules before others, especially in selectors that have lots of rules.

Reply:
We don't have any kind of consistency or even internal guidelines around property ordering at present. Ideally we would use a tool (maybe [stylelint](https://github.com/stylelint/stylelint)) or auto-formatter to enforce as appropriate, but even some kind of agreed convention would be a first step.